packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownVariables.ts

```diff
@@ -65,7 +65,7 @@ export interface DropdownVariables {
   loadingMessageBackgroundColor: string;
 }
 
-const [cornerRadius, _12px_asRem] = [3, 12].map(v => pxToRem(v));
+const [cornerRadius, _12px_asRem] = [4, 12].map(v => pxToRem(v));
```

Comment:
maybe have some reference to a global variable here as well?

Reply:
I think this is just converting 4 & 12 to rems, so I'll remove this and update the single variables.